eo-maven-plugin/src/main/java/org/eolang/maven/MjParse.java

```diff
@@ -146,12 +149,26 @@ private Node parsed(final Path source, final String identifier) throws IOExcepti
         );
         final String name = new ObjectName(xmir).get();
         if (!name.equals(identifier)) {
-            throw new IllegalArgumentException(
-                String.format(
-                    "Tojo identifier '%s' does not match to result object name '%s'",
-                    identifier, name
-                )
-            );
+            try {
+                new Xembler(
+                    new Directives()
+                        .xpath("/object")
+                        .addIf("errors")
+                        .add("error")
+                        .attr("check", "validate-object-name")
+                        .attr("severity", "critical")
+                        .set(
+                            String.format(
+                                "Tojo identifier '%s' does not match to result object name '%s'",
+                                identifier, name
+                            )
+                        )
+                ).apply(xmir.inner());
```

Comment:
@h1alexbel let's move `xmir.inner()` to a separated variable and use it here and in `return` statement